src/main/java/com/microsoft/spring/data/gremlin/query/query/QueryFindScriptGenerator.java

```diff
@@ -36,13 +37,24 @@ private String generateIsEqual(@NonNull Criteria criteria) {
             subject = PROPERTY_ID; // If subject is @Id/id field, use id property in database.
         }
 
-        final String has = AbstractGremlinScriptLiteral.generateHas(subject, criteria.getSubValues().get(0));
+        final String content = AbstractGremlinScriptLiteral.generateHas(subject, criteria.getSubValues().get(0));
 
-        return String.format(GREMLIN_PRIMITIVE_WHERE, has);
+        return String.format(GREMLIN_PRIMITIVE_WHERE, content);
+    }
+
+    private String generateAfter(@NonNull Criteria criteria) {
+        final String subject = criteria.getSubject();
+        final long milliSeconds = GremlinUtils.timeToMilliSeconds(criteria.getSubValues().get(0));
```

Comment:
Is there any check that the type of the value in the criteria is time type, or it is not necessary?

Reply:
Yes, I only allowed type `Date` here, it may support more in future. Also add instance check for that 
in GremlinUtils. See following code:
```java
    public static long timeToMilliSeconds(@NonNull Object time) {
        if (time instanceof Date) {
            return ((Date) time).getTime();
        } else {
            throw new UnsupportedOperationException("Unsupported time type");
        }
    }
```